net/wifi-presence/Makefile

```diff
@@ -0,0 +1,63 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=wifi-presence
+PKG_VERSION:=0.1.2
+PKG_RELEASE:=1
+
+PKG_SOURCE:=-$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/awilliams/wifi-presence/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=f5cf8bf36e3c17ad4b0486007532030760c22c5f63bd63707d911ab94bd744df
+
+PKG_LICENSE:=MIT
+PKG_MAINTAINER:=Adam Williams <pwnfactory@gmail.com>
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+PKG_USE_MIPS16:=0
+
+GO_PKG:=github.com/awilliams/wifi-presence
+GO_PKG_BUILD_PKG:=$(GO_PKG)/cmd/wifi-presence
+
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/golang/golang-package.mk
+
+define Package/wifi-presence
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=WiFi presence detection
+  URL:=https://github.com/awilliams/wifi-presence
+  DEPENDS:=$(GO_ARCH_DEPENDS)
+endef
+
+define Package/wifi-presence/description
```

Comment:
I appreciate that this package is compatible with any MQTT broker, but it might be worth mentioning HomeAssistant here so that it can be found easier for those looking for HA integrations.

Reply:
Yes, good point. Updated to `WiFi presence detection with Home Assistant integration`. Sound OK?